docs/standards/hcs-8/operations.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Operations
+sidebar_position: 2
+---
+
+#### [ hcs-8 - Poll Topic Standard ]
+
+The following operations are defined:
+
+| Name        | Operation string (case sensitive) |
+|-------------|-----------------------------------|
+| Register    | register                          |
+| Manage      | manage                            |
+| Vote        | vote                              |
+| Update      | update                            |
+| Information | information                       |
+
+---
+
+### Register
+
+The Register operation must be the first message in the topic. Topics without a valid register operation as the first message are not considered valid HCS-8 polls. 
+
+The metadata for the register operation should be a JSON-formatted string. Poll metadata is recommended to follow hcs9: Poll Metadata Standard to allow for cross-platform compatibility.
+
+When the Register metadata exceeds the maximum length of a HCS message (1024 characters), follow up Register operations are posted, as many as needed to complete the JSON string.
+
+Only the first message in the topic, or the first set of messages (as identified by the ‘sid’ field) are legal register operations. Any register operations that occur after this are invalid and ignored.
```

Comment:
optional, consider storing full-json as an HCS-1 topic and referencing utilizing a CDN and `t_id` similar to other standards e.g. 

https://github.com/hashgraph-online/hcs-improvement-proposals/pull/19/files#diff-9e49559cb51bc730827cc515a0c37eaede4cbce543f20ad4688acc2a00dc960fR192

reference to the `standards-sdk` for inscribing files: https://github.com/hashgraph-online/standards-sdk?tab=readme-ov-file#usage